src/commands/ImportObjectsCommandGenerator.js

```diff
@@ -59,50 +59,59 @@ module.exports = class ImportObjectsCommandGenerator extends BaseCommandGenerato
 	}
 
 	async _getCommandQuestions(prompt) {
+						
+		let listObjectAnswers;
 		try {
 			const listObjectQuestions = this._generateListObjectQuestions();
-			const listObjectAnswers = await prompt(listObjectQuestions);
-
-			const paramsForListObjects = this._arrangeAnswersForListObjects(listObjectAnswers);
-			const executionContextForListObjects = new SDKExecutionContext({
-				command: this._listObjectsMetadata.name,
-				params: paramsForListObjects,
-				includeProjectDefaultAuthId: true,
-			});
+			listObjectAnswers = await prompt(listObjectQuestions);
```

Comment:
Do not surround this with a try/catch block. If there is an error reading the manifest file (change the name to manifestQQ,xml), the code won't show an appropriate message.